packages/addon-dev/src/rollup-app-reexports.ts

```diff
@@ -25,21 +26,21 @@ export default function appReexports(opts: {
           this.emitFile({
             type: 'asset',
             fileName: `_app_/${appFilename}`,
-            source: `export { default } from "${pkg.name}/${addonFilename.slice(
-              0,
-              -extname(addonFilename).length
-            )}";\n`,
+            source: `export ${opts.defaultExport} from "${
```

Comment:
This is going to lead to very confusing failures when somebody puts the wrong thing in the opts.

I don't think we should ask the user to supply a fully-valid snippet of javascript that happens to work in this context, I think they should give us a `string[]` of exports and we should emit the correct javascript.